### ISSUE TITLE
Fix the jest-ts tests 

### ISSUE THREAD

**Issue by krassowski** [krassowski/jupyterlab-go-to-definition#8]
I was not able to pinpoint the issue but there might be a need to use babel or tweak the transform in another way to make it work. Help would be greatly appreciated!

**Comment by krassowski** [comment]
Example of a failing test:

```js
> @krassowski/jupyterlab_go_to_definition@0.3.0 test /home/travis/build/krassowski/jupyterlab-go-to-definition
> npx jest
 FAIL  src/languages/python.spec.ts
  ● Test suite failed to run
    Jest encountered an unexpected token
    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.
    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".
    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.
    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html
    Details:
    /home/travis/build/krassowski/jupyterlab-go-to-definition/node_modules/@jupyterlab/codeeditor/lib/index.js:3
    export * from './editor';
    ^^^^^^
    SyntaxError: Unexpected token export
      1 | import { expect } from 'chai';
      2 | 
    > 3 | import { CodeEditor } from "@jupyterlab/codeeditor";
        | ^
      4 | import { CodeMirrorEditor } from '@jupyterlab/codemirror';
      5 | 
      6 | import { CodeMirrorTokensProvider } from "../editors/codemirror/tokens";
      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:471:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:513:25)
      at Object.<anonymous> (src/languages/python.spec.ts:3:1)

```

**Comment by krassowski** [comment]
Fixed with 4bbcaa6664581ac996756170201a6283c11bb9cf.
